.github/workflows/test-collectibles.yml

```
@@ -0,0 +1,36 @@
+      - "config/**"
+      - ".github/**"
+      - "pnpm-lock.yaml"
+
+env:
+  SCARB_VERSION: v2.9.2
+
+jobs:
+  test-collectibles:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)